service/examples/README.md

```diff
@@ -13,13 +13,7 @@ $ GOOS=linux GOARCH=mips GOMIPS=softfloat go build hello.go -o /tmp/zkm/test/hel
 If you build your own server, you can use Docker Compose (The image is built based on AMD64)
 
 Minimum Requirements
-| SEG_SIZE | RAM |
-| ------- | ------- |
-| 1024 | 16G |
-| 16384 | 28.2G |
-| 32768 | 95.2G |
-| 65536 | 96.3G |
-| 262144 | 130.1G |
+At least 64GB of memory is required
```

Comment:
Just remove this requirement.  Or we can say 4GB is required.

Reply:
okey